src/include/sof/library_manager.h

```diff
@@ -0,0 +1,159 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2022 Intel Corporation. All rights reserved.
+ *
+ * Author: Jaroslaw Stelter <jaroslaw.stelter@intel.com>
+ *         Pawel Dobrowolski <pawelx.dobrowolski@intel.com>
+ */
+
+/*
+ * Library manager implementation for SOF is using codec adapter API for
+ * loadable native and external libraries. Depends of information received from
+ * it manages memory and locate accordingly to available space.
```

Comment:
`codec adapter` is no longer used. Everything is called `module adapter` now. Also you are alternating `library_manager` and `lib_manager` in your code/file names.

I would suggest sticking to `lib_manager` in all places.